apps/emqx_bridge_hstreamdb/test/emqx_bridge_hstreamdb_SUITE.erl

```diff
@@ -13,8 +13,13 @@
 -include_lib("snabbkaffe/include/snabbkaffe.hrl").
 
 % SQL definitions
--define(STREAM, "stream").
+
+-define(STREAM, "demo_steram").
+%% could not be "stream" in Production Environment
```

Comment:
is there some checking for that?

Reply:
Currently no, even hstreamdb_erl can create a stream named `stream`